### DISCUSSION TITLE
[Node Operator Question] Getting "rpc method is not whitelisted" in debug module

### DISCUSSION THREAD

**Discussion by jsjsjsbzbz** [ethereum-optimism/developers#505] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi, 

So I am running an archive node following this guide https://docs.optimism.io/builders/node-operators/rollup-node

After having the node synced up, I wanted to check some historical data such as tracing some oldest blocks. However it seems that I need to whitelist some methods to be able to do such request. Otherwhise I get the following error:

![image](https://github.com/ethereum-optimism/developers/assets/169898811/fa8d5887-6f6d-42d4-8a6b-9c2ba738a623)

Is there any documentation about how I can whitelist all methods? 

### Protocol Description

op-geth Geth/v1.101315.2-stable-7c281983/linux-amd64/go1.21.1

**Comment by imtipi** [comment]
how you run your node and can you post your start command?

**Reply by jsjsjsbzbz** [reply]
Following the script used by the compose file for the `op-geth`service in https://github.com/smartcontracts/simple-optimism-node:

```
 exec geth \
  --datadir="$BEDROCK_DATADIR" \
  --http \
  --http.corsdomain="*" \
  --http.vhosts="*" \
  --http.addr=0.0.0.0 \
  --http.port=8545 \
  --http.api=web3,debug,eth,txpool,net,engine \
  --ws \
  --ws.addr=0.0.0.0 \
  --ws.port=8546 \
  --ws.origins="*" \
  --ws.api=debug,eth,txpool,net,engine,web3 \
  --metrics \
  --metrics.influxdb \
  --metrics.influxdb.endpoint=http://influxdb:8086 \
  --metrics.influxdb.database=opgeth \
  --syncmode="$OP_GETH__SYNCMODE" \
  --gcmode="$NODE_TYPE" \
  --authrpc.vhosts="*" \
  --authrpc.addr=0.0.0.0 \
  --authrpc.port=8551 \
  --authrpc.jwtsecret=/shared/jwt.txt \
  --rollup.sequencerhttp="$BEDROCK_SEQUENCER_HTTP" \
  --rollup.disabletxpoolgossip=true \
  --port="${PORT__OP_GETH_P2P:-39393}" \
  --discovery.port="${PORT__OP_GETH_P2P:-39393}" \
  --db.engine=pebble \
  --state.scheme=hash 
  ```

**Reply by imtipi** [reply]
> Following the script used by the compose file for the `op-geth`service in https://github.com/smartcontracts/simple-optimism-node:
> 
> ```
>  exec geth \
>   --datadir="$BEDROCK_DATADIR" \
>   --http \
>   --http.corsdomain="*" \
>   --http.vhosts="*" \
>   --http.addr=0.0.0.0 \
>   --http.port=8545 \
>   --http.api=web3,debug,eth,txpool,net,engine \
>   --ws \
>   --ws.addr=0.0.0.0 \
>   --ws.port=8546 \
>   --ws.origins="*" \
>   --ws.api=debug,eth,txpool,net,engine,web3 \
>   --metrics \
>   --metrics.influxdb \
>   --metrics.influxdb.endpoint=http://influxdb:8086 \
>   --metrics.influxdb.database=opgeth \
>   --syncmode="$OP_GETH__SYNCMODE" \
>   --gcmode="$NODE_TYPE" \
>   --authrpc.vhosts="*" \
>   --authrpc.addr=0.0.0.0 \
>   --authrpc.port=8551 \
>   --authrpc.jwtsecret=/shared/jwt.txt \
>   --rollup.sequencerhttp="$BEDROCK_SEQUENCER_HTTP" \
>   --rollup.disabletxpoolgossip=true \
>   --port="${PORT__OP_GETH_P2P:-39393}" \
>   --discovery.port="${PORT__OP_GETH_P2P:-39393}" \
>   --db.engine=pebble \
>   --state.scheme=hash 
> ```

have you run l2geth?and how you set OP_GETH__HISTORICAL_RPC= ?
